README.md

```diff
@@ -0,0 +1,126 @@
+# spark-rewards
+
+This implements revision D of [Meridian: Off-chain scheduled rewards](https://spacemeridian.notion.site/Meridian-Off-chain-scheduled-rewards-f9480ef009464ecfaf02a4f752cc4d70).
+
+## Routes
+
+### `GET /scheduled-rewards`
+
+Response:
+
+```js
+{
+  "address1": "scheduledRewards1",
+  "address2": "scheduledRewards2",
+  // ...
+}
+```
+
+### `GET /scheduled-rewards/:address`
+
+Response:
+
+```js
+"scheduledRewards1"
+```
+
+### `POST /scores`
+
+Request:
+
+```js
+{
+  "participants": {
+    "address1": "score1",
+    "address2": "score2"
+  },
+  "signature": {
+    "r": "...",
+    "s": "...",
+    "v": "..."
+  }
+}
+```
+
+Sign over a packed keccak256 with this schema:
+
+```js
+['address[]', 'uint256[]']
+```
+
+Response:
+
+```js
+{
+  "updatedAdress1": "scheduledRewards1",
+  "updatedAddress2": "scheduledRewards2",
+  // ...
+}
+```
+
+### `POST /paid`
+
+Request:
+
+```js
+{
+  "participants": {
+    "address1": "amount1",
+    "address2": "amount2",
+    // ...
+  },
+  "signature": {
+    "r": "...",
+    "s": "...",
+    "v": "..."
+  }
+}
+```
+
+Sign over a packed keccak256 with this schema:
+
+```js
+['address[]', 'uint256[]']
+```
+
+Response:
+
+```js
+{
+  "adddress1": "scheduledRewards1",
+  "address2": "scheduledRewards2",
+  // ...
+}
+```
+
+### `GET /log`
```

Comment:
I have decided to add this out-of-spec functionality, so that we have an audit trail of all changes that happened to the data.